docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -791,6 +791,11 @@ given module
 -   `${<module_name>_owner}` is the Github repo owner for the evergreen
     module associated with this task
 
+In the [Github merge queue](Merge-Queue.md), a single additional expansion
+called `${github_head_branch}` is available. This is the name of the temporary
+branch that GitHub creates for this merge group item. It looks something like
+"gh-readonly-queue/main/pr-515-9cd8a2532bcddf58369aa82eb66ba88e2323c056".
```

Comment:
Should we add the note here about the fact that "This might not make sense when
multiple builds are being tested together, but it will work for them most of the
time"? 
What is it that wont work and should we maybe try to ensure that we are not creating an expectation that it wont meet?

Reply:
done